src/LLVM_Runtime_Linker.cpp

```diff
@@ -692,8 +691,11 @@ std::unique_ptr<llvm::Module> get_initial_module_for_target(Target t, llvm::LLVM
                 // TODO: Replace fake thread pool with a real implementation.
                 modules.push_back(get_initmod_fake_thread_pool(c, bits_64, debug));
             } else if (t.os == Target::NoOS) {
-                // No externally resolved symbols are allowed here.
-                modules.push_back(get_initmod_noos(c, bits_64, debug));
+                // The OS-specific symbols provided by the modules
+                // above are expected to be provided by the containing
+                // process instead at link time. Less aggressive than
+                // NoRuntime, as OS-agnostic modules like tracing are
```

Comment:
I wonder if NoRuntime should also be an OS enumeration value, since NoRuntime implies the choice of OS is irrelevant... right? Probably not worth changing at this point though (and either way, not in this PR).